cvdupdate/cvdupdate.py

```diff
@@ -255,12 +255,12 @@ def update(self, db: str = "") -> bool:
 
     def clean_dbs(self):
         """
-        Delete all files in the database directory.
+        Delete cvd controlled files in the database directory.
```

Comment:
It appears this PR is based off of the branch including the change to only delete f"{self.db_dir / db}"`, so I will be unable to merge it until that PR is fixed and this one is based off of the fixed branch, or off of `main`.